optional_plugins/README.rst

```diff
@@ -0,0 +1,26 @@
+==================
+ Optional Plugins
+==================
+
+This is where plugins shipped with Avocado, but not considered core
```

Comment:
This is confusing... We have core and plugins. Now we have plugins considered core.
I think we should not use the word 'core' for plugins. Instead, we should have `required` and `optional` plugins.